src/libcommonserver/io/iohelper.cpp

```diff
@@ -591,6 +591,15 @@ bool IoHelper::checkIfPathExistsWithSameNodeId(const SyncPath &path, const NodeI
             LOGW_WARN(logger(), L"Error in IoHelper::getNodeId for path=" << Path2WStr(path).c_str());
         }
         exists = (nodeId == tmpNodeId);
+
+#ifdef NDEBUG
```

Comment:
Shouldn't we send the sentry only from the the DeleteJob, along side with the debug message "still exist on local replica. Aborting current sync and restart."? Because `checkIfPathExistsWithSameNodeId` is called in others places and we don't want too many false positive sentry messages.

Reply:
To discuss...
Whatever the solution there will be false positives.
We can add a 2nd Sentry in the DeleteJob.

Reply:
Seconding Clément, I would let the `Sentry` message be as close as possible to the potential issue it aims at warning on. 
The reason is that, without additional context, `exists = false` is just a normal situation for the function `getNodeId`.